app/javascript/components/SelectCountry/SelectCountry.js

```diff
@@ -47,17 +47,8 @@ export class SelectCountry extends Component {
     let countries = countriesByLocale[props.intl.locale];
 
     countries = Object.keys(countries)
-      .map(c => {
-        return { value: c, label: countries[c] };
-      })
-      .sort((a, b) => {
-        let countryA = a.label.toUpperCase();
-        let countryB = b.label.toUpperCase();
-        if (countryA < countryB) return -1;
-        if (countryA > countryB) return 1;
-
-        return 0;
-      });
+      .map(c => ({ value: c, label: countries[c] }))
+      .sort(new Intl.Collator(props.intl.locale).compare);
```

Comment:
Seems like this makes `flow` complain. `window.Intl.Collator...` might fix it? (to specify that it's not in a node env, but a browser env).

Reply:
👍